CHANGELOG.md

```diff
@@ -16,11 +17,19 @@
   * Add `cardThreeDSecureDidCancel()` to `CardVaultDelegate`
   * Add `cardThreeDSecureWillLaunch()` to `CardVaultDelegate`
   * Add `cardThreeDSecureDidFinish()` to `CardVaultDelegate`
+  * Add PrivacyInfo.xcprivacy file
 * PayPalWebPayments
   * Add `vault(url:)` method to `PayPalWebCheckoutClient`
   * Add `PayPalVaultResult` type to return vault result
   * Add `PayPalVaultDelegate` to handle results from vault flow
   * Add `PayPalWebCheckoutClientError.paypalVaultResponseError` for missing or invalid response from vaulting
+  * Add PrivacyInfo.xcprivacy file
+* CorePayments
+  * Add PrivacyInfo.xcprivacy file
+* PayPalNativePayments
+  * Add PrivacyInfo.xcprivacy file
+* FraudProtection
+  * Add PrivacyInfo.xcprivacy file
```

Comment:
Take it or leave it comment - IMO one changelog entry for all the modules is fine for the PrivacyManifest additions!

Reply:
I think previously for Apple mandated things we've also linked to an Apple post with the requirements, so we can also consider adding something like [meets Apple's new Privacy Update requirements](https://developer.apple.com/news/?id=3d8a9yyh) - the same syntax should work in the changelog for links as here, so can do: `[my very cool text here](www.myurl.com)`